### ISSUE TITLE
ivdata/snapclient linux/arm64/v8 arch support

### ISSUE THREAD

**Issue by ChristianRiesen** [IVData/dockerfiles#28]
Hi. First of all thanks, your images are nice and clean, just switched to them and ran into a snag when trying to start the snapclient on a raspberry pi 4. 
```
# docker pull ivdata/snapclient
Using default tag: latest
latest: Pulling from ivdata/snapclient
no matching manifest for linux/arm64/v8 in the manifest list entries
```

I see there is a `linux/arm64/v7` there and hope it wouldn't be too hard to add it for `v8` as well.

**Comment by hamishfagg** [comment]
Hey, thanks =)

This is a bit more complicated than it would first appear. The snapcast project doesn't provide an arm64 package, so I'd have to build it.

This might be a good time to switch over to using alpine as well to make the images smaller. I will work on this, just not sure how long it'll take as I have other priorities.

**Comment by hamishfagg** [comment]
I've just pushed an alpine image to `ivdata/snapclient:alpine` which has an arm64 version. Give it a try and let me know if it works :)

**Comment by ChristianRiesen** [comment]
Thanks a lot @IVData

I tried it and it pulled immediately, but now I have no function. It starts up and after about 10 - 14 seconds it goes into a status of `Restarting (0) x seconds ago` on `docker ps` output. I never see it crop up in the snapcast server. I tried coaxing it to output more logs by trying different log sinks and output metadata to stderr but it is at all times just silent. Also `docker logs` shows me nothing, ever.

Any idea what I could do to help you debug it?

The container runs fine on a raspberry pi 3 with buster. This one I'm trying to get it to run on is a raspberry pi 4 with ubuntu server (20.04.4).

This is my start command:
```
docker run -d \
   --name snapclient \
   --restart=unless-stopped \
   --device /dev/snd \
   -e HOST=192.168.1.151 \
   -e EXTRA_ARGS='--hostID Livingroom -s sndrpihifiberry --logsink=system' \
   ivdata/snapclient:alpine
```
